src/layouts/Layout.astro

```diff
@@ -151,44 +149,14 @@ const bannerOffset = bannerOffsetByPosition[siteConfig.banner.position || 'cente
 		<div id="page-height-extend" class="hidden h-[300vh]"></div>
 	</body>
 </html>
+
 <style is:global>
 	:root {
 		--hue: var(--configHue);
 		--page-width: 75rem;
 	}
 </style>
-<style is:global define:vars={{ bannerOffset }}>
```

Comment:
This line defines the CSS variable `--bannerOffset`. Removing this causes it to become undefined. I am not sure but this might be the reason why the banner isn't working.

Reply:
resolve